.gitignore

```diff
@@ -1 +1 @@
-.vscode
```

Comment:
I think we still need to maintain our .gitignore or just add new items. lets leave .vscode there.